src/main/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ add_executable(collab-vm-server
 target_include_directories(collab-vm-server PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(collab-vm-server PRIVATE ${CMAKE_THREAD_LIBS_INIT} Boost::program_options Boost::filesystem ${CMAKE_DL_LIBS} spdlog::spdlog fmt::fmt collabvm::core collabvm::websocket)
 
-if(WIN32)
+if(WIN32 OR MSYS OR MINGW)
```

Comment:
I mean mingw enviroments would just shortcircuit when WIN32 is true anyways

Reply:
makes no sense in that context but apparently some cross-compiling envs specify either MSYS or MINGW and *not* WIN32 for some reason.